mqtt/mqtt-bridge/src/lib.rs

```diff
@@ -8,7 +8,8 @@
     clippy::use_self,
     clippy::match_same_arms,
     clippy::must_use_candidate,
-    clippy::missing_errors_doc
+    clippy::missing_errors_doc,
+    dead_code
```

Comment:
Since the persistor/loader/waking_state isn't used yet, is this ```dead_code``` necessary?

Reply:
I saw this pattern in Anca's PR, but I can add this allow to the top of specific files. Any objections?

Reply:
lets move it in modules that contain dead code instead? please put a TODO to not to forget remove it later